.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: csharp
+solution: Mono.Cecil.sln
+install:
+  - nuget restore Mono.Cecil.sln
```

Comment:
(more likely than Cecil using a NuGet package would be to grab NUnit as a NuGet package in UTs, it'll be required then as well)